Formula/flann.rb

```diff
@@ -1,9 +1,9 @@
 class Flann < Formula
   desc "Fast Library for Approximate Nearest Neighbors"
   homepage "https://www.cs.ubc.ca/research/flann/"
-  url "https://github.com/mariusmuja/flann/archive/1.9.1.tar.gz"
-  sha256 "b23b5f4e71139faa3bcb39e6bbcc76967fbaf308c4ee9d4f5bfbeceaa76cc5d3"
-  revision 9
+  url "https://github.com/xlingsky/flann/archive/m1.9.1.tar.gz"
+  sha256 "acd1e95cdf6e942061a4a4d373ff921c96062018736a4a448a5f2ca9024c8fd2"
+  revision 10086
```

Comment:
Why not 10?

Reply:
Yes. It should be 10. This value is just for testing